salt/edx/prod.sls

```diff
@@ -224,3 +224,23 @@ add_{{ host.name }}_to_known_hosts_for_edxapp:
     - user: www-data
     - fingerprint: {{ host.fingerprint }}
 {% endfor %}
+
+configure_nginx_status_module_for_edx:
+  file.managed:
+    - name: /etc/nginx/sites-enabled/status_monitor
+    - contents: |
+        server {
+            listen 80;
+            location /nginx_status {
+                stub_status on;
+                access_log off;
+                allow 127.0.0.1;
+                deny all;
```

Comment:
Correct, but it's also best practice from a security perspective to not expose this endpoint to external clients without having it locked down.